Users/Glob_Patterns.rst

```diff
@@ -192,13 +182,48 @@ brackets. Parentheses that have no match are ignored as well as
 
 ::
 
-    >>> fnmatch("abc", "a**c")
-    True
     >>> fnmatch("abbc", "a**c")
     True
     >>> fnmatch("a/c", "a**c")
     True
-    >>> fnmatch("a?c", "a**c")
-    True
-    >>> fnmatch("ac", "a**c")
-    True
+
+Using Glob Patterns
+-------------------
```

Comment:
this paragraph is nice, let's put it on top so users see it and then can see the "advanced example collection" if it doesn't suffice?